vm/src/stdlib/operator.rs

```diff
@@ -217,9 +218,9 @@ mod _operator {
 
     /// Return the number of occurrences of b in a.
     #[pyfunction(name = "countOf")]
-    fn count_of(a: PyObjectRef, b: PyObjectRef, vm: &VirtualMachine) -> PyResult<usize> {
+    fn count_of(a: PyIter, b: PyObjectRef, vm: &VirtualMachine) -> PyResult<usize> {
         let mut count: usize = 0;
-        let iter = iterator::get_iter(vm, a)?;
+        let iter = a;
```

Comment:
seems (here and in `index_of`) like these aren't necessary anymore (the type gives enough info on what it is already)